scripts/extensions/sams/src/components/workspaceSubnav.tsx

```diff
@@ -179,86 +189,147 @@ export class WorkspaceSubnav extends React.PureComponent<IProps> {
         const buttonIcon = this.props.currentSet?.state === SET_STATE.DISABLED ? 'lock' : undefined;
         const sortFieldText = getAssetListSortFieldText(this.props.searchParams.sortField);
 
-        return (
-            <React.Fragment>
-                <SubNav zIndex={2}>
-                    <ButtonGroup align="inline">
-                        <Dropdown items={items}>
+        if (this.selectedAssets()) {
+            return (
+                <React.Fragment>
+                    <SubNav zIndex={2}>
+                        <div className="multi-action-bar">
+                            <button className="toggle" onClick={this.props.closeMultiActionBar}>
+                                <i className="icon-chevron-up-thin" />
+                            </button>
+                            <button className="btn" onClick={this.props.closeMultiActionBar}>cancel</button>
+                            <span id="multi-select-count">{this.props.selectedAssetIds?.length} Item selected</span>
```

Comment:
Use `gettextPlural` to flag translation. Example `gettextPlural` usage:
https://github.com/superdesk/superdesk-client-core/blob/sams/scripts/extensions/sams/src/containers/FileUploadModal.tsx#L202